h/search/__init__.py

```diff
@@ -11,6 +12,7 @@
     'TopLevelAnnotationsFilter',
     'get_client',
     'init',
+    'connect'
```

Comment:
I am fond of this naming—we're establishing a connection here, not "creating a client." It also helps to make more clear the fact that each invocation of `connect` will, you know, create a new connection.

Reply:
:+1: